src/validation.cpp

```diff
@@ -5468,6 +5480,7 @@ bool ChainstateManager::PopulateAndValidateSnapshot(
 
     bool out_of_coins{false};
     try {
+        COutPoint outpoint;
```

Comment:
I think this should be:

```cpp
Txid txid;
coins_file >> txid;
```

The current code might accidentally work because a `COutPoint` is a `Txid` followed by a single number (albeit `uint32_t` instead of `uint16_t`).

Reply:
Should be addressed in #29612